platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitService.java

```diff
@@ -391,6 +388,17 @@ public void tagDelete(TagDeleteRequest request) throws ApiException {
                     String[] keyValues = row.split("=", 2);
                     result.put(keyValues[0], keyValues[1]);
                 }
+            } else if (request.isSet()) {
```

Comment:
Please rework current `getConfig` method to a post api method with query parameter and add a new `getConfig` method as get api method with `ConfigRequest` DTO.